schema/patches/10.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+CREATE TABLE public.short_urls (
+    short_url_id SERIAL PRIMARY KEY,
+    slug TEXT NOT NULL,
+    redirect_to TEXT NOT NULL
+);
+
+CREATE TABLE public.short_url_clicks (
+    click_id BIGSERIAL PRIMARY KEY,
+    short_url_id INTEGER REFERENCES public.short_urls (short_url_id) ON DELETE CASCADE,
+    click_time TIMESTAMPTZ NOT NULL,
+    user_agent TEXT DEFAULT NULL,
+    ip_address INET DEFAULT NULL
+);
+
+INSERT INTO public.l_action (descr, phpconstant) VALUES ('Edit Short URLs', 'AUTH_EDITSHORTURLS');
```

Comment:
could this be added in `schema/whichever` file so its with all the others, and all of those get added when someone makes a local myradio. and we can just add this one ourselves to prod

Reply:
Oh yeah that exists, will change